deptofdefense/solo
# Fixed local HTTPS/CAC Authentication to work with the Terraform AWS Deployment Changes

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist
I have…
- ~~[ ] review and update SSAG documentation if needed.~~

## Summary of Changes
This pull request…
- closes #114 
- enables developers to locally run a HTTPS/CAC enabled build/iterations, and streamlines the setup with a script.

## Testing
To verify the changes proposed in this pull request…
1. Pull down the branch
2. Follow the instructions to CONTRIBUTING.md
3. verify the website is accessible.
## Screenshots
~~_Attach relevant before and after screenshots here._~~
